.github/workflows/tests.yml

```diff
@@ -18,6 +18,7 @@ jobs:
 
     env:
       MPICH_INTERFACE_HOSTNAME: localhost
+      JULIA_MPIEXEC_TEST_ARGS: "-hosts localhost"
```

Comment:
@jakebolewski this seems to make it work

Reply:
ok good to know, we'll have to have to add something similar to clima

Reply:
probably needed both to avoid the fallback behavior :-/